Modules/_csv.c

```diff
@@ -1072,7 +1072,7 @@ csv_reader(PyObject *module, PyObject *args, PyObject *keyword_args)
         return NULL;
     }
 
-    if (!PyArg_UnpackTuple(args, "", 1, 2, &iterator, &dialect)) {
+    if (!PyArg_UnpackTuple(args, "_csv.reader", 1, 2, &iterator, &dialect)) {
```

Comment:
This unnecessary exposes the implementation detail (the private module name).
```pycon
>>> import csv
>>> csv.reader
<built-in function reader>
>>> csv.reader()
Traceback (most recent call last):
  File "<python-input-2>", line 1, in <module>
    csv.reader()
    ~~~~~~~~~~^^
TypeError: _csv.reader expected at least 1 argument, got 0
```
Compare with:
```pycon
>>> import os
>>> os.mkdir()
Traceback (most recent call last):
  File "<python-input-1>", line 1, in <module>
    os.mkdir()
    ~~~~~~~~^^
TypeError: mkdir() missing required argument 'path' (pos 1)
```

Reply:
Yes, this was my open question. I will remove the module name.

@hauntsaninja sorry, I should have make this more clear :(